### ISSUE TITLE
Can't install component: no_unconfigured_devices

### ISSUE THREAD

**Issue by FredericoCBarbosa** [pbutterworth/nissan-leaf-obd-ble#5]
Hi

I'm getting an error when i try to install this integration:  "no_unconfigured_devices"

Any tips?

Fred

**Comment by pbutterworth** [comment]
Hi @FredericoCBarbosa,
I don't recognise that error. Can you please supply some more information?
Suggested info:
* screenshots of the error, and any steps leading up to it
* log file snippets, if relevant

**Comment by TommySharpNZ** [comment]
I can get this error when I use the "Add Integration" and manually select "Nissan Leaf OBD BLE".
I did try this because the ODB dongle is never discovered by HA so I thought I'd give it a try.

**Comment by FredericoCBarbosa** [comment]
Yes, I can confirm that the issue occurs when I try to activate the integration in Home Assistant.

![Clipboard01](https://github.com/user-attachments/assets/31c67759-2c99-4928-9b93-0e714f68cfd7)
 
The logs don't show any relevant issue, only this warning: 

Registo: homeassistant.core
Origem: helpers/deprecation.py:222
Primeira ocorrência: 19:35:26 (1 ocorrências)
Último registo: 19:35:26

Config was used from nissan_leaf_obd_ble, this is a deprecated alias which will be removed in HA Core 2025.11. Use homeassistant.core_config.Config instead, please report it to the author of the 'nissan_leaf_obd_ble' custom integration

**Comment by pbutterworth** [comment]
@FredericoCBarbosa 
OK, I see what is going on.
This integration doesn't work by manually adding it. It must be discovered (and therefore can store the address of the device). 
What type of OBD dongle are you using?

**Comment by FredericoCBarbosa** [comment]
Some bootleg Chinese dongle that works with LeafSpy. I don't know the brand, but I've modified the UUID in the elm327.py and manifest.json files. The UUID is somewhat similar to the default one: 00001101-0000-1000-8000-00805f9b34fb.

**Comment by pbutterworth** [comment]
Is that the service UUID?
You might need to also change the characteristic UUIDs. And check to see whether there is a separate UUID for read and write.

**Comment by FredericoCBarbosa** [comment]
I can only see one UUID for the whole device. So, it will not work, right?

**Comment by FredericoCBarbosa** [comment]
Ok, I’ve discovered that this particular dongle uses Bluetooth BR/EDR technology, not Bluetooth LE. What a bummer.
